mirascope/core/cohere/call_kwargs.py

```diff
@@ -0,0 +1,13 @@
+"""This module contains the type definition for the Cohere call keyword arguments."""
+
+from cohere.types import ChatMessage
+
+from mirascope.core.base.call_kwargs import BaseCallKwargs
+
+from .call_params import CohereCallParams
+from .tool import CohereTool
+
+
+class CohereCallKwargs(CohereCallParams, BaseCallKwargs[CohereTool]):
+    model: str
+    message: ChatMessage
```

Comment:
I believe `message` is just a`str`, not a `ChatMessage`

Reply:
Oh, I made mistake.

Reply:
Fixed